scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -210,6 +216,19 @@ def discover(self):
                     logger.debug(f'Quarantine file {quarantine_file} is empty')
             self.quarantine = Quarantine(ql)
 
+        #handle extra scripts
+        sl = self.options.scripting_list
+        if sl:
+            for scripting_file in sl:
+                try:
+                    # validate scripting yaml file against the provided schema
+                    scl.yaml_load_verify(scripting_file, self.scripting_schema)
+                except scl.EmptyYamlFileException:
+                    logger.debug(f'Scripting file {scripting_file} is empty')
+                except FileNotFoundError:
+                    logger.error(f'Scripting file {scripting_file} not found')
```

Comment:
shouldn't twister stop executing and exit in this point ?

Reply:
it should, added